inc/cli/class-assign-defaults-command.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * HM Utility Taxonomy CLI: Assign default terms to existing published posts
+ */
+
+declare( strict_types=1 );
+
+namespace HM\Utility_Taxonomy\CLI;
+
+use HM\Utility_Taxonomy as HMUT;
+use WP_CLI;
+use WP_Post;
+use WP_Query;
+
+/**
+ * Assign default terms
+ *
+ * @since 1.4.0
+ */
+class Assign_Defaults_Command {
+	/**
+	 * Number of found posts
+	 *
+	 * @since 1.4.0
+	 *
+	 * @var int
+	 */
+	protected $found_posts = 0;
+
+	/**
+	 * Maximum number of pages
+	 *
+	 * @since 1.4.0
+	 *
+	 * @var int
+	 */
+	protected $max_pages = 0;
+
+	/**
+	 * Current paged number
+	 *
+	 * @since 1.4.0
+	 *
+	 * @var int
+	 */
+	protected $paged = 1;
+
+	/**
+	 * Default posts per page
+	 *
+	 * @since 1.4.0
+	 *
+	 * @var int
+	 */
+	protected $posts_per_page = 100;
+
+	/**
+	 * Used to check whether we're dry-running or not
+	 *
+	 * @since 1.4.0
+	 *
+	 * @var bool
+	 */
+	protected $is_dry_running = false;
+
+	/**
+	 * Map term IDs to term names
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param array $term_ids Array of term IDs.
+	 *
+	 * @return array
+	 */
+	protected function term_ids_to_names( array $term_ids ) : array {
+		return array_map( function ( int $id ) : string {
+			return get_term( $id, HMUT\TAXONOMY )->name;
+		}, $term_ids );
+	}
+
+	/**
+	 * Process post
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param WP_Post $post Post object.
+	 *
+	 * @return void
+	 */
+	protected function process( WP_Post $post ) {
```

Comment:
The function can have a return type void:
```suggestion
	protected function process( WP_Post $post ) : void {
```